### ISSUE TITLE
Playstation games not installing into Openemu

### ISSUE THREAD

**Issue by Therealmadkong** [OpenEmu/OpenEmu#1977]
I have been using the cd-based games guide, but am still having trouble.  I have ape escape, Parappa the rapper and policenauts english patch version, but they won't install.  Ape escape came as a .zip  so I unarchived it, and it is a folder now with only a .bin.  I cut the .bin part off the title and put in .cue.  Openemu installs it, but nothing comes up on the games list.  Policenauts came as a Policenauts_Eng.zip so I unarchived it and the folder holds disc 1 .mdf and .mds, and a disc 2 .mdf and .mds.  I cut off the .mdf of each and replaced it with .cue, but nothing happend.  Parappa the rapper was different.  It came in .rar so I used UnrarX and got a .ccd file.  The guide said these would work so I put it openemu and nothing happened.  I have the Playstation bios ready to go, but I cannot get it to work.  Thank you for your time.

**Comment by clobber** [comment]
Please give me a step by step list of what exactly you are doing and what isn't working, because it isn't so clear to me. Be precise. It'd also be nice to have a list of all your files and contents of your cue sheets pasted back here.

It sounds to me that you have simply renamed a .bin file to a .cue, and that won't work.

**Comment by Therealmadkong** [comment]
You are right on the money I renamed the .bin file to a .cue file.  I have not opened a cue sheet, just changed the name.  I had to do this with the Bios for it to work so I thought it would work here.The only thing I did before that was get a .zip of the rom and unarchived it.  .mdf and .mds were the only files for Policenauts avaliable since it included the english language patch.  I am guessing those files don't work, because patches cannot be inclued with the game.  I watched a tutorial here: https://www.youtube.com/watch?v=4ncTG2Vq8bQ and used the CD based guide for open emu.  So i know that it only takes .cue or .ccd files, but Parappa the rapper had a .ccd file and it still didn't install.
